ds3_java_cli/src/main/java/com/spectralogic/ds3cli/exceptions/CommandExceptionFactory.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * *****************************************************************************
+ *   Copyright 2014-2016 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ***************************************************************************
+ */
+
+package com.spectralogic.ds3cli.exceptions;
+
+import com.spectralogic.ds3client.networking.FailedRequestException;
+import java.io.IOException;
+
+public class CommandExceptionFactory {
+
+    /**
+     * Provide an exception with a common message "[Command class] failed: reason, then exception
+     *
+     * @param commandName pass in simple name of command class or string describing module
+     * @param e the caught exception
+     * @return a new Command Exception with regularized message.
+     */
+
+    public static CommandException getResponseExcepion(final String commandName, final IOException e) {
```

Comment:
I don't know that the name of the first argument reflects that is actually a commandName.  You also pass in other exception types.